scrapers_next/ct/people.py

```diff
@@ -0,0 +1,91 @@
+from spatula import CsvListPage
+from openstates.models import ScrapePerson
+
+
+class LegList(CsvListPage):
+    source = "ftp://ftp.cga.ct.gov/pub/data/LegislatorDatabase.csv"
+
+    def process_item(self, row):
+        name = row["first name"]
+        mid = row["middle initial"].strip()
+        if mid:
```

Comment:
if you could replace this old style string formatting with new that'd be great for clarity

either

name += " " + row["last name"]

or name += f" {row['last name']}"